BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -33,6 +33,30 @@ extension StackElement {
         configure(&self)
     }
 
+    /// Initializer using result builder to declaritively build up a stack.
+    /// - Parameters:
+    ///   - elementsBuilder: A block containing all elements to be included in the stack.
+    /// - Note: If element is a StackChild, then traits and key will be pulled from the element, otherwise defaults are passed through.
+    public init(@ElementBuilder<StackChild> elementsBuilder: () -> [StackChild]) {
+        self.init()
+        children = elementsBuilder().map { stackChild in
+            (stackChild.element, stackChild.traits, stackChild.key)
+        }
+    }
+
+    /// Initializes stack transforming an array of some item into an array of elements.
+    /// - Parameters:
+    ///   - items: The array of items to transform.
+    ///   - transform: Closure to be applied to each item `T` which transforms that item into an `Element`.
+    ///   - configure: A closure used to modify the stack.
+    public init<T>(_ items: [T], transform: (T) -> StackChild, configure: (inout Self) -> Void = { _ in }) {
+        self.init()
+        children = items.map(transform).map { stackChild in
+            (stackChild.element, stackChild.traits, stackChild.key)
+        }
+        configure(&self)
+    }
```

Comment:
Do these need to be public?

Reply:
Probably not since Row and Column have inits with all default values. Also, I don't think the second one is necessary at all anymore since I added the ability for result builders to take in arrays of children. The same thing can be accomplished with a simple `map` on the items you are trying to convert to Elements.